gui/gui/src/ts/components/frame/Frame.tsx

```diff
@@ -20,7 +20,9 @@ export const Frame: React.FunctionComponent<{children: React.ReactNode}> = ({
   return (
     <div data-cy="frame">
       <Navbar></Navbar>
-      <div className={classes.root}>{children}</div>
+      <div className={classes.root} data-cy="body">
```

Comment:
Nit: the name body is a bit generic here, frame-content or similar would be better.

Reply:
Makes sense, changed